package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Tanzu Developer Center",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "postinstall": "husky install"
```

Comment:
You can add a script to this block here.

```
"scripts": {
  "sort": "node sort-dict/sort-dict.js custom_dict.txt"
}
```

Scripts are always assumed to be run from the directory containing package.json (ie the project root), so now you don't need to deal with `dirname`, and your hook simply invokes it with `npm run sort`.

You can also do away with the `.husky` directory, and add this to package.json:

```
"husky": {
  "hooks": {
    "pre-commit": "npm run sort"
  }
}
```

This might be preferable because it keeps all the logic in on eplace.